src/content/about/using-cache.md

```diff
@@ -3,143 +3,39 @@ title: Using the Cache
-
-The Cache API's `caches.default` Cache object allows you to:
-
-- Fetch a response for a URL if (and only if) it is cached using `caches.default.match(..)`.
-
-- Explicitly store a response in the cache using `caches.default.put(..)` and explicitly delete `caches.default.delete(..)`.
-
-**What are the usage limits on the cache API?**
-
-- 50 total `put()`, `match()`, or `delete()` calls per-request, using the same quota as `fetch()`
-
-- 5 GBs total `put()` per-request
+**What should I use: the Cache API or fetch for caching objects on Cloudflare? **
+For requests where Workers are behaving as middleware (i.e. they are sending a subrequest via `fetch`) it is recommended to use fetch. This is because pre existing settings are in place that optimize caching while preventing unintended dynamic caching. For projects where there is no backend (i.e. the entire project is on Workers as in [Workers Sites](/sites)) the Cache API is the only option for caching.
 
-- For Free, Pro, Business:
+### `fetch`
 
-  - up to 512MB for each `put()` with a valid `Content-Length` header
-
-  - up to 512MB for a `put()` with `Transfer-Encoding: chunked`. Note that this blocks subsequent `put()`s until the transfer completes.
-
-- For Enterprise:
-
-  - up to 5GBs for each `put()` with a valid `Content-Length` header
-
-  - up to 5GBs for a `put()` with `Transfer-Encoding: chunked`. Note that this blocks subsequent `put()`s until the transfer completes.
-
-  Using the Cache API will _not_ set a `Cf-Cache-Status` header.
-
-### Cloudflare's CDN
-
-Cloudflare's traditional CDN will work by default on static assets even without Workers. The namespace used by the Cache API is stored separate from Cloudflare's CDN though both exist on Cloudflare's Global Network.
-
-In the context of Workers a [`fetch`](/reference/apis/fetch) provided by the runtime communicates with the traditional Cloudflare cache. First, `fetch` checks to see if the URL matches a different zone. If it does, it reads through that zone's cache. Otherwise, it reads through its own zone's cache, even if the URL is for a non-Cloudflare site. Cache settings on `fetch` automatically apply caching rules based on your Cloudflare settings. `fetch` does not allow you to _modify or inspect objects_ before they reach the cache, but does allow you to modify _how it will cache_.
+In the context of Workers a [`fetch`](/reference/apis/fetch) provided by the runtime communicates with the Cloudflare cache. First, `fetch` checks to see if the URL matches a different zone. If it does, it reads through that zone's cache. Otherwise, it reads through its own zone's cache, even if the URL is for a non-Cloudflare site. Cache settings on `fetch` automatically apply caching rules based on your Cloudflare settings. `fetch` does not allow you to _modify or inspect objects_ before they reach the cache, but does allow you to modify _how it will cache_.
```

Comment:
"... it reads through that zone's cache."

Maybe add, "(or worker!)."?

(I'm trying to win an award for my punctuation on that one.)